src/transformers/models/lxmert/modeling_tf_lxmert.py

```diff
@@ -953,6 +932,7 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFLxmertModelOutput,
         config_class=_CONFIG_FOR_DOC,
     )
+    @unpack_inputs
```

Comment:
In other PRs, `@unpack_inputs` is the top decorator. Do you mind moving it to the top? 🙏  (as well as in the other uses of `@unpack_inputs`)